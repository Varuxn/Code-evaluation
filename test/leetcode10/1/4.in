93 159
4 39
64 87
30 59
54 49
15 27
3 5
22 70
27 90
33 75
32 67
32 78
66 80
40 28
70 89
83 37
79 34
21 76
26 42
65 34
73 21
13 14
22 67
88 28
55 8
4 84
37 77
7 71
42 44
17 26
2 69
34 37
58 91
57 20
24 30
26 60
12 46
1 3
2 20
0 16
85 31
11 28
26 82
18 24
12 50
16 65
67 85
69 35
2 40
6 45
54 57
10 13
32 21
66 38
79 81
68 92
12 53
88 29
57 80
12 88
5 58
0 9
35 62
24 66
38 28
58 35
22 54
6 40
81 67
43 66
22 30
57 79
74 86
36 52
13 46
9 18
23 81
72 4
33 83
25 38
70 76
62 29
6 7
0 2
89 7
0 12
84 41
45 61
37 27
45 54
16 32
41 72
12 47
83 45
49 89
5 74
37 58
22 29
5 23
6 8
81 29
9 10
57 39
10 72
50 15
14 55
3 17
21 41
5 68
27 91
42 77
24 23
9 31
24 48
4 33
2 4
21 56
84 77
20 34
55 59
66 47
65 73
47 35
1 11
2 27
0 1
92 12
63 28
8 56
51 53
28 47
2 25
24 33
17 22
19 81
17 35
87 88
6 9
31 36
81 62
61 62
76 83
15 19
37 43
10 21
2 74
61 63
77 75
14 71
35 64
24 26
3 6
41 74
6 51
41 49
0 15
18 48
80 54
33 37
21 15
