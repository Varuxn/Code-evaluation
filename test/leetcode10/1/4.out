26 60 26 82 27 90 31 36 32 78 36 52 42 44 74 86 